cvat/apps/dataset_manager/annotation.py

```diff
@@ -199,22 +199,27 @@ def clear_frames(self, frames: Container[int]):
             # Tracks are not expected in the cases this function is supposed to be used
             raise AssertionError("Partial annotation cleanup is not supported for tracks")
 
-    def to_shapes(self,
+    def to_shapes(
+        self,
         end_frame: int,
         *,
         included_frames: Optional[Sequence[int]] = None,
         include_outside: bool = False,
-        use_server_track_ids: bool = False
+        use_server_track_ids: bool = False,
+        deleted_frames: Optional[Collection[int]] = None,
```

Comment:
I think it's supposed to be handled by `included_frames`. Could you check if this parameter can be reused for deleted frames as well?

Reply:
Reused this parameter. I had an apparently wrong impression that, on job export, tracks from previous jobs of the same task are exported as well